plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -62,6 +63,7 @@ def associate(source_repo, dest_repo, import_conduit, config, units=None, solver
     units = set(units)
     failed_units = set()
     # we need to filter out rpm units since in reality they were not associated
+    # they are associated in one batch later, so _associate_unit simply ignores them
```

Comment:
> _associate_unit simply ignores them
Unfortunately, `_associate_unit` doesn't just ignore them, it returns the unit back without associating it. This is the main reason we need to filter it out explicitly here.
Maybe ` # they are associated in one batch later` is enough. What do you think?

Reply:
Sounds good